### PR TITLE
Changing Minimum and Maximum times for the nuclear charge

### PR DESCRIPTION
**Pull request by AmazingDragon353**
## About the PR 

This changes the nuclear charge's (the one activated using nukeman) minimum and maximum time from 30 and 440 to 120 and 600 seconds respectively. It also sets the default time to 180 seconds. 

## Why's this needed? 

This is because setting off a nuke on a 30 second timer is currently near impossible to defuse, and people deserve a chance to deal with it. It also increases the maximum time to give you a little more wiggle room if you want to rp a bomb threat without it accidentally going off. Its maximum time would also end up being equal to the nuclear operatives gamemode's nuke timer, for consistency's sake.

## Changelog

```changelog
(u)AmazingDragons
(+)Changes the minimum, maximum, and default times (in seconds) on the nuclear charge to 120, 600, and 180 respectively.
```

### CODE REVIEW

**Review by Tarmunora — CHANGES_REQUESTED** [review]
cleaner to use `clamp(value, low, high)` instead of `max(min(value, high), low)`

**Comment by AmazingDragon353** [comment]
> cleaner to use `clamp(value, low, high)` instead of `max(min(value, high), low)`

Done!

File: code/modules/networks/computer3/mainframe2/misc_terms.dm [hunk 1]
```diff
@@ -1219,7 +1219,8 @@
 //					SPAWN_DBG(0.3 SECONDS)
 //						src.post_status(target, "command","term_disconnect")
 //					return
-
+#define MIN_NUKE_TIME 120
+#define MAX_NUKE_TIME 600
```
Comment (Tarmunora) [inline code/modules/networks/computer3/mainframe2/misc_terms.dm:1223]: move to `_std/defines/computer.dm`

**Review by Tarmunora — CHANGES_REQUESTED** [review]

File: code/modules/networks/computer3/mainframe2/misc_terms.dm [hunk 2]
```diff
@@ -1144,6 +1142,9 @@
 				outpost_destroyed = 1
 				src.detonate()
 				return
+			if(src.time == DISARM_CUTOFF)
+				src.icon_state = "net_nuke2"
+				boutput(world, "<span class='alert'><b>[src.time] seconds until nuclear charge detonation.</b></span>")
```
Comment (Tarmunora) [inline code/modules/networks/computer3/mainframe2/misc_terms.dm:1147]:
```suggestion
```
snip

File: code/modules/networks/computer3/mainframe2/os_drivers.dm [hunk 3]
```diff
@@ -1064,7 +1064,7 @@
 				if (!isnum(data["time"]))
 					return ESIG_GENERIC
 
-				var/newtime = max(0, min(data["time"], 440))
+				var/newtime = max(0, min(data["time"], MAX_NUKE_TIME))
```
Comment (Tarmunora) [inline code/modules/networks/computer3/mainframe2/os_drivers.dm:1067]: could use clamp here too

**Review by Tarmunora — CHANGES_REQUESTED** [review]

**Review by Tarmunora — APPROVED** [review]

**Pull request merged**